src/azure-cli/azure/cli/command_modules/appservice/appservice_environment.py

```diff
@@ -140,34 +141,23 @@ def create_ase_inbound_services(cmd, resource_group_name, name, subnet, vnet_nam
     if not ase:
         raise ResourceNotFoundError("App Service Environment '{}' not found.".format(name))
 
+    if ase.internal_load_balancing_mode == 'None':
+        raise ValidationError('Private DNS Zone is not relevant for External ASE.')
+
+    if ase.kind.lower() == 'asev3':
+        # pending SDK update (ase_client.get_ase_v3_networking_configuration(resource_group_name, name))
+        raise CommandNotFoundError('create-inbound-services is currently not supported for ASEv3.')
+
+    ase_vip_info = ase_client.get_vip_info(resource_group_name, name)
+    inbound_ip_address = ase_vip_info.internal_ip_address
     inbound_subnet_id = _validate_subnet_id(cmd.cli_ctx, subnet, vnet_name, resource_group_name)
     inbound_vnet_id = _get_vnet_id_from_subnet(cmd.cli_ctx, inbound_subnet_id)
-    if ase.kind.lower() == 'asev3':
-        _ensure_subnet_private_endpoint_network_policy(cmd.cli_ctx, inbound_subnet_id, False)
-        network_client = _get_network_client_factory(cmd.cli_ctx)
-        pls_connection = PrivateLinkServiceConnection(private_link_service_id=ase.id,
-                                                      group_ids=['hostingEnvironments'],
-                                                      request_message='Link from CLI',
-                                                      name='{}-private-connection'.format(name))
-        private_endpoint = PrivateEndpoint(location=ase.location, tags=None, subnet=Subnet(id=inbound_subnet_id))
-        private_endpoint.private_link_service_connections = [pls_connection]
-        poller = network_client.private_endpoints.begin_create_or_update(resource_group_name,
-                                                                         '{}-private-endpoint'.format(name),
-                                                                         private_endpoint)
-        LongRunningOperation(cmd.cli_ctx)(poller)
-        ase_pe = poller.result()
-        nic_name = parse_resource_id(ase_pe.network_interfaces[0].id)['name']
-        nic = network_client.network_interfaces.get(resource_group_name, nic_name)
-        inbound_ip_address = nic.ip_configurations[0].private_ip_address
-    elif ase.kind.lower() == 'asev2':
-        if ase.internal_load_balancing_mode == 0:
-            raise ValidationError('Private DNS Zone is not relevant for External ASEv2.')
-        ase_vip_info = ase_client.get_vip_info(resource_group_name, name)
-        inbound_ip_address = ase_vip_info.internal_ip_address
```

Comment:
Why remove support for `asev3`. Will this cause breaking change to users?

Reply:
The use of private endpoint for ASEv3 was only relevant during preview. For GA we moved to using internal Azure load balancer and therefore the section is not relevant anymore. This command is now only an additional help to setup private DNS zone if Cx would like that. For ASEv3 we have to wait for an update of the SDK to 3.0.0 before we can support this.
So nothing broken. And the command is still flagged as preview.

Reply:
Got it, thanks~